c/common/test/rules/functionnoreturnattributecondition/test.c

```diff
@@ -77,7 +77,7 @@ _Noreturn void test_noreturn_f10(int i) { // COMPLIANT
   case 4:
     thrd_exit(0);
     break;
-  default:
```

Comment:
Is this semicolon intentional?

Reply:
Yes, a label has to occur before a statement, and a variable declaration doesn't count as a statement. Obscure fun!